applications/crossbar/priv/api/swagger.json

```diff
@@ -36,6 +36,7 @@
             "type": "object"
         },
         "account_config.cdr": {
+            "additionalProperties": false,
```

Comment:
Hey so this is weird https://github.com/json-schema-org/json-schema-spec/blob/f7bb080b0c6af0b23505cdcc0143bccf001b6d40/jsonschema-validation.xml#L509